src/main/pg/motor.h

```diff
@@ -59,7 +59,7 @@ typedef struct motorDevConfig_s {
 
 typedef struct motorConfig_s {
     motorDevConfig_t dev;
-    uint16_t idleOffset;                    // Idle offset value for motor protocols, full motor output = 10000
+    uint16_t motorIdle;                    // Idle offset value for motor protocols, full motor output = 10000
```

Comment:
Is this more descriptive?
```suggestion
    uint16_t motorIdle;                    // Value for idle throttle of motor protocols, full motor output = 10000
```

Reply:
used 
`// When motors are at idle, the amount added above the disarmed value, in percent * 100 of the motor range.`